README.md

```diff
@@ -12,6 +12,15 @@
   <a href="https://github.com/GetStream/stream-chat-android/releases"><img src="https://img.shields.io/github/v/release/GetStream/stream-chat-android" /></a>
 </p>
 
+<div align="center">
+
+![stream-chat-android-client](https://img.shields.io/badge/stream--chat--android--client-3.01%20MB-lightgreen)
+![stream-chat-android-offline](https://img.shields.io/badge/stream--chat--android--offline-3.22%20MB-lightgreen)
+![stream-chat-android-ui-components](https://img.shields.io/badge/stream--chat--android--ui--components-7.88%20MB-lightgreen)
+![stream-chat-android-compose](https://img.shields.io/badge/stream--chat--android--compose-9.70%20MB-lightgreen)
+
+</div>
```

Comment:
Should we include some "note" about this SDK size notifying it is the max size our SDK will include to their app on the hypothetical case their app is not using any of the dependencies our SDK uses (Almost impossible, because lot of them would need to be used)?